dev/ci/ci-hott.sh

```diff
@@ -7,4 +7,4 @@ HoTT_CI_DIR=${CI_BUILD_DIR}/HoTT
 
 git_checkout ${HoTT_CI_BRANCH} ${HoTT_CI_GITURL} ${HoTT_CI_DIR}
 
-( cd ${HoTT_CI_DIR} && ./autogen.sh && ./configure && make -j ${NJOBS} )
+( cd ${HoTT_CI_DIR} && ./autogen.sh && ./configure && ./etc/coq-scripts/timing/make-pretty-timed-or-error.sh -j ${NJOBS} && make validate )
```

Comment:
*nods*  I'd be happy to have it merged into Coq, though I'm not sure where it should go.  It's [a pretty simple python script](https://github.com/JasonGross/coq-scripts/blob/master/timing/TimeFileMaker.py); basically, it parses the output of one or two `make TIMED=1`, associates the time(s) to file names, takes the differences if there are two text blobs you pass it, sorts them, and prints them.

Where do you think it should go / how should it be called?